src/components/HslaStringColorPicker.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import { AlphaColorPicker } from "./common/AlphaColorPicker";
+import { ColorModel, ColorPickerBaseProps } from "../types";
+import { equalColorString } from "../utils/compare";
+import { hslaStringToHsva, hsvaToHslaString } from "../utils/convert";
+
+const colorModel: ColorModel<string> = {
+  defaultColor: "hsl(0, 0%, 0%, 1)",
```

Comment:
Looks like `hsvaToHslaString` uses `hsla()` format, but the default value here is in `hsl` format.